src/algoliasearch.helper.js

```diff
@@ -959,7 +958,7 @@ AlgoliaSearchHelper.prototype.isRefined = function(facet, value) {
  *
  */
 AlgoliaSearchHelper.prototype.hasRefinements = function(attribute) {
-  if (!isEmpty(this.state.getNumericRefinements(attribute))) {
+  if (Object.keys(this.state.getNumericRefinements(attribute)).length > 0) {
```

Comment:
this.state.getNumericRefinements(attribute) seems to be an array
Redundant, but adding it since it's not using `objectHasKeys` like the rest.

Reply:
looks like it was forgotten to be refactored